mods/voidmulti.sc2mod/base.sc2data/gamedata/weapondata.xml

```diff
@@ -112,9 +117,16 @@
         <MinScanRange value="5.5"/>
         <TargetFilters value="Ground,Visible;Missile,Stasis,Dead,Hidden,Invulnerable"/>
     </CWeaponLegacy>
+    <CWeaponStrafe id="InterceptorBeam">
+        <LoiterInnerRadius value="3"/>
+        <LoiterRadius value="5"/>
+    </CWeaponStrafe>
     <CWeaponLegacy id="JavelinMissileLaunchers">
         <MinScanRange value="10.5"/>
     </CWeaponLegacy>
+    <CWeaponLegacy id="KaiserBlades">
+        <RangeSlop value="1.4"/>
```

Comment:
Ultralisk's attack's range slop increased from 1.0 to 1.4 (= the range a target can move away before the attack is cancelled

This makes it better vs units running away since they need to move 0.4 further away.
Usually, units have this value set to 1.0